auto_dev/data/repo/templates/autonomy/install.sh

```diff
@@ -127,26 +139,27 @@ function install_poetry_deps() {
     local pip_executable
     local poetry_executable
     local host_poetry_executable
-    host_poetry_executable=$(echo -n $(poetry env info | grep Executable |head -n 1 | awk -F: '{ print $2 }') | xargs)
-    echo "Host poetry executable: $host_poetry_executable"
+    host_poetry_executable="$(poetry env info | grep Executable | head -n 1 | awk -F: '{ print $2 }' | xargs)"
+    echo "Host poetry executable: ${host_poetry_executable}"
     echo "Setting up new poetry environment..."
 
-    poetry env use $(which python)
-    poetry_executable=$(echo -n $(poetry env info | grep Executable |head -n 1 | awk -F: '{ print $2 }') | xargs)
-    echo "New poetry executable:   $poetry_executable"
-
+    python_path="$(command -v python)"
+    poetry env use "${python_path}"
```

Comment:
which at this point should be returning system python no?
Does which not work on fedora?

Reply:
I'm in the venv I created for this package so it returns .venv/bin/python
without reading every line of your install script and understanding poetry (which I've never used before) I wouldn't trust your `make install` to run in my global environment. I wouldn't encourage people to run anything outside of a venv.
looks like if someone ran this outside of a venv they would get their global python yeah

Reply:
reverted here 79afc73e5c86ed64a8afa8c4da1870b7ba80c036

Reply:
turns out the `which` was giving this error:

```
Setting up new poetry environment...
/usr/bin/env: ‘node’: No such file or directory
```

sw itching from `which` to `command` fixes it in 4d457b36f8e28bd46ede294627e91b49609c91f4

Reply:
Sweet, command also works so it seems to be good